conf/messages

```diff
@@ -49,6 +49,9 @@ vatDetails.title = Your VAT account
 vatDetails.manageYourVat = Manage your VAT
 vatDetails.vatCertificate.view = View and print your VAT certificate.
 vatDetails.vatCertificate.link = View VAT certificate
+vatDetails.vatOptOut.view = You cannot opt out if your taxable turnover has been above £85,000 since 1 April 2019.
+vatDetails.vatOptOut.link = Opt out of Making Tax Digital for VAT
+vatDetails.vatOptOut.error = Sorry, there is a problem with the service. Try again later.
```

Comment:
Do we have Welsh content for this message? If not at the moment that's fine as we discussed not blocking stories anymore due to Welsh, but could you check if there is a story to do this work, and if not raise a new one?